stl/inc/future

```diff
@@ -141,6 +138,11 @@ private:
     error_code _Mycode; // the stored error code
 };
 
+// TRANSITION, ABI: remove `_v2` suffix

```

Comment:
Conventionally we just append a version number to the symbol name without a `_v` delimiter. I don't mind changing that convention, as long as we are doing so consciously and this isn't a one-off.

Reply:
I was thinking of the `_Stodx_v2()` precedent:

https://github.com/microsoft/STL/blob/314f65f831f827dbfb05ebcc0f08c496f5ddf64f/stl/inc/xlocnum#L46

You're right that there's much more precedent for plain `2`, although it's less prominent (there are many non-"version 2" uses of 2).

As I'm removing the original declaration, there's no risk of calling the original, and I can't argue that the longer suffix is buying anything. I'll change it, thanks.